README.rst

```diff
@@ -42,14 +42,32 @@ path to the SQLite database.
 
     $ xbrl_extract {path_to_filings} {path_to_database}
 
+This repo contains a small selection of FERC Form 1 filings from 2021 in the
+``examples`` directory. To test the tool on these filings, use the command
+
+.. code-block:: console
+
+    $ xbrl_extract examples/ferc1-2021-sample.zip {path_to_database}
```

Comment:
Can you go ahead and include a dummy value for `path_to_database` here for clarity, e.g.

```
$ xbrl_extract examples/ferc1-2021-sample.zip ./ferc1-2021-sample.sqlite
```

Is this a database which has to already exist? Or is this a new database that will be created? What happens if a database already exists at that location?

Reply:
Added output path